src/test/mod.rs

```diff
@@ -2394,5 +2394,72 @@ fn batch_insert_1() {
     assert_eq!(res[1].big, Some(-3));
 }
 
+#[test]
+#[ignore]
+fn large_returning() {
+    let mut conn = init_testing();
+    create_gst_types_table(&mut conn);
+
+    let res = gst_types::table.load::<GSTTypes>(&mut conn).unwrap();
+
+    assert_eq!(res.len(), 0);
+    const SIZE: usize = 40_000;
+    let large_string = std::iter::repeat('a').take(SIZE).collect::<String>();
+    let large_bin = std::iter::repeat(1).take(SIZE).collect::<Vec<u8>>();
+
+    let res = ::diesel::insert_into(gst_types::table)
+        .values(Newgst_types::new(
+            None,
+            None,
+            None,
+            None,
+            Some(large_string.clone()),
+            Some(large_bin.clone()),
+            None,
+            None,
+            Some("bac".into()),
+        ))
+        .returning((gst_types::text, gst_types::byte, gst_types::v))
+        .get_result::<(Option<String>, Option<Vec<u8>>, Option<String>)>(&mut conn)
+        .unwrap();
+
+    assert_eq!(res.0, Some(large_string));
+    assert_eq!(res.1, Some(large_bin));
+    assert_eq!(res.2, Some("bac".into()));
+
+    let res2 = gst_types::table.get_result::<GSTTypes>(&mut conn).unwrap();
+    assert_eq!(res.0, res2.text);
+    assert_eq!(res.1, res2.byte);
+    assert_eq!(res.2, res2.v);
+}
+
+// #[test]
+// fn explicit_single_returning() {
```

Comment:
Do we want this test or not? If the first, it should be included, otherwise remove it completely.